dobi.yaml

```diff
@@ -6,3 +6,73 @@ image=image-kas:
   image: ghcr.io/siemens/kas/kas
   tags: ['2.3.3']
   pull: once
+
+image=image-curl-jq:
+  image: dwdraju/alpine-curl-jq
+  tags: [latest]
+  pull: once
+
+image=image-mender-cli:
+  image: ci4rail/mender-cli
+  tags: [master]
+  pull: once
+
+# ===================================================
+# mounts
+# ===================================================
+
+# Where the yocto-build happens
+mount=build-dir-yocto:
+  bind: yocto
+  path: /work
+
+# The location of kasfile.yaml (kas expects it in /repo)
+mount=repo-dir-yocto:
+  bind: yocto
+  path: /repo
+  read-only: true
+
+# Yocto downloads DL_DIR
+mount=download-dir-yocto:
+  bind: "{env.YOCTO_DOWNLOAD_DIR:downloads}"
+  path: /downloads
+
+# Yocto shared state cache SSTATE_DIR
+mount=sstate-dir-yocto:
+  bind: "{env.YOCTO_SSTATE_CACHE_DIR:}yocto/sstate-cache"
```

Comment:
is this correct? @see mount=download-dir-yocto

Reply:
Yes, it is correct. 
All these things are not new. They have been there since month...

Reply:
TODO: Move default sstate to cache/downloads (don't rely on trailing slash)